frontend/src/data/variables/AcsPopulationProvider.ts

```diff
@@ -73,7 +105,7 @@ class AcsPopulationProvider extends VariableProvider {
     const enabledBreakdown = Object.values(
       breakdowns.demographicBreakdowns
     ).find((breakdown) => breakdown.enabled === true)!;
-    df = applyToGroups(df, ["state_name"], (group) => {
+    df = applyToGroups(df, [geoNameColumn], (group) => {
```

Comment:
Hmm, I think this should be fipsColumn? I think it was done this way because early datasets might not have fips columns but I think they all do now